src/pages/community/meetups.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Druid Community Meetups
+layout: simple_page
+canonical: 'https://druid.apache.org/community/meetups'
+---
+
+## Druid Community Meetups
+
+Check out [Apache Druid on meetup.com](https://www.meetup.com/topics/apache-druid/) for links to regular
+meetups in cities all over the world.
+
+All Apache Druid Meetups are covered the Apache [anti-harassment policy](https://www.apache.org/foundation/marks/events#anti-harassment) and speakers are expected to follow the guidelines below.
+
+### Speaker Guidelines
+
+**Skip the Sales Pitch:** Keep your talk unbiased and educational. It’s okay to mention commercial products you’re using, but only if they are a key part of your data architecture and Apache Druid. Content which is solely focused on promotion of a product or service is never permitted at an Apache Druid Meetup.
```

Comment:
I think we need to be explicit that it's ok to mention things being possible in the free versions of products, and to emphasise that education - rather than persuasion - is the priority.

Reply:
Oooh and also vendor neutrality - not sure how we fit this in tho?

Reply:
@petermarshallio what aspects of vendor neutrality are important to cover here?